native/screens/Applicants/Applicants.jsx

```diff
@@ -0,0 +1,91 @@
+import { Button, Text, TouchableOpacity, ScrollView, View } from 'react-native';
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import uuid from 'react-native-uuid';
+import { handleToggle } from '../../services/utils/handlers';
+
+export const Applicants = ({ navigation }) => {
+  const soloProject = useSelector((state) => state.projects.allProjects[0]);
+  const reduxUserID = useSelector((state) => state.ui.user._id);
+  const [engineers, setEngineers] = useState([]);
+  const [designers, setDesigners] = useState([]);
+  const [verifiedOwner, toggleVerifiedOwner] = useState(false);
+
+  useEffect(() => {
+    if (soloProject && soloProject.owner === reduxUserID) {
+      toggleVerifiedOwner(true);
+    } else {
+      toggleVerifiedOwner(false);
+      return;
+    }
+
+    const filterRoles = (role) => {
+      return soloProject.interested_applicants?.filter((applicant) => applicant.role === role);
+    };
+
+    setEngineers(() => filterRoles('Software Engineer'));
+    setDesigners(() => filterRoles('UX Designer'));
+  }, [reduxUserID, soloProject]);
+
+  if (verifiedOwner) {
+    return (
+      <ScrollView>
+        <Text>Applications</Text>
+        <RoleList applicants={engineers} navigation={navigation} role={'Software Developers'} />
+        <Text>{'\nJUST A DIVIDER \n DONT MIND ME \n DELETE ME AFTER \n'}</Text>
+        <RoleList applicants={designers} navigation={navigation} role={'UX Designers'} />
+      </ScrollView>
+    );
+  } else {
+    return <Text> LOADING OR UNAUTHORIZED </Text>;
+  }
+};
+
+const RoleList = ({ applicants, navigation, role }) => {
+  const [loadMore, toggleLoadMore] = useState(false);
+  const [visibleList, setVisibleList] = useState([]);
+
+  useEffect(() => {
+    if (!applicants) {
+      return;
+    } else if (loadMore) {
+      setVisibleList(applicants);
+    } else {
+      setVisibleList(applicants.slice(0, 3));
+    }
+  }, [loadMore, applicants]);
+
```

Comment:
I like this approach for the 'load more' functionality. Can't wait to see it in action!